editor/scripts/bundle.py

```diff
@@ -283,7 +283,7 @@ def extract_build_jdk(build_jdk):
 def invoke_lein(args, jdk_path=None):
     # this weird dance with env and bash instead of supplying env kwarg to run.command is needed for the build script to work on windows
     jdk_path = jdk_path or os.environ['JAVA_HOME']
-    return run.command(['env', 'JAVA_CMD=%s/bin/java' % jdk_path, 'LEIN_HOME=target/lein', 'bash', './scripts/lein'] + args)
+    return run.command(['env', 'JAVA_CMD=%s/bin/java' % jdk_path, 'LEIN_HOME=build/lein', 'bash', './scripts/lein'] + args)
```

Comment:
I moved lein install to `build` because some lein commands clear the `target` dir, which causes the CI to re-download lein multiple times per run.